### ISSUE TITLE
[Feature Request] Add Notes

### ISSUE THREAD

**Issue by avnery4** [bTayFla/PtTracker#12]
I think it will be very useful to add a note in some point in time, to track configuration changes that we've made or anything else thats worth noting. Maybe on the daily row?

**Comment by nolnet** [comment]
I agree you could add it here see screen shot => https://www.screencast.com/t/NXc5yt0nj

**Comment by Tornado-Truxton** [comment]
Good idea and I agree . . .

**Comment by clownfish44** [comment]
A button to add notes should be added to the daily as well as per trade.
